apps/ledger-live-desktop/src/renderer/families/polkadot/NominationFlowModal/steps/StepConfirmation.tsx

```diff
@@ -23,7 +24,35 @@ const Container = styled(Box).attrs(() => ({
 }>`
   justify-content: ${p => (p.shouldSpace ? "space-between" : "center")};
 `;
-function StepConfirmation({ t, optimisticOperation, error, signed }: StepProps) {
+function StepConfirmation({
+  t,
+  optimisticOperation,
+  error,
+  signed,
+  transaction,
+  source,
+}: StepProps) {
+  const preloaded = usePolkadotPreloadData();
+  const { validators: allValidators } = preloaded;
+
+  const validators = useMemo(() => {
+    return allValidators
+      .filter(val => transaction.validators.includes(val.address))
+      .map(val => val.identity || val.address);
+  }, [allValidators, transaction.validators]);
```

Comment:
[Question] Do we not want to return 1 validator or undefined and not a string[]?